tty_windows.go

```diff
@@ -0,0 +1,25 @@
+//go:build windows
+// +build windows
+
+package main
+
+import (
+	"strings"
+	"strconv"
+	"github.com/shirou/gopsutil/host"
+)
+
+func defaultShellWithArgs() []string {
+	hostInfo, _ := host.Info()
```

Comment:
I don't think you actually need `github.com/shirou/gopsutil/host` module because the same can be done with `golang.org/x/sys/windows`'s `RtlGetNtVersionNumbers` function.

Reply:
I tend to agree, we can probably do this without extra deps.